collectors/ebpf.plugin/ebpf.c

```diff
@@ -1061,12 +1079,17 @@ static void read_collector_values(int *disable_apps)
     }
     *disable_apps = (int)enabled;
 
+    // Cgroup is a positive sentence, so we need to invert the values to disable apps.
+    // We are using the same pattern for cgroup and apps
+    enabled = appconfig_get_boolean(&collector_config, EBPF_GLOBAL_SECTION, EBPF_CFG_CGROUP, CONFIG_BOOLEAN_YES);
+    *disable_cgroups =  (enabled == CONFIG_BOOLEAN_NO)?CONFIG_BOOLEAN_YES:CONFIG_BOOLEAN_NO;
```

Comment:
WE will need to apply this to the new `shm` charts as well just merged.

Reply:
I added. Thanks!